klaus/highlighting.py

```diff
@@ -34,17 +36,18 @@ def _format_lines(self, tokensource):
 
     def _lookup_ctag(self, token):
         matches = list(self._get_all_ctags_matches(token))
-        best_matches = self.get_best_ctags_matches(matches)
-        if not best_matches:
+        try:
+            best_match = next(self.get_best_ctags_matches(matches))
```

Comment:
> Well using next() will be faster since it means we can stop before we have the complete list. To be honest, I'd have used it for matches too :)

I'd say this sounds like premature optimization, doesn't it? If you're interested in benchmarking the performance difference (could be a simple as trying to notice a difference in server response time), you're welcome; otherwise, I'd always propose to go with the unoptimized, simpler version of some piece of code.

> But as it's your module, I'll use the list approach. I'll repush in a few minutes.

I appreciate the thought but I think we should try to keep personal preference out of such discussions :-)